### PR TITLE
create wikidata language module

### PR DESCRIPTION
**Pull request by mr-martian**
This module scrapes language population and region data from Wikidata since it is no longer possible to get this from Ethnologue (#513).

### CODE REVIEW

File: modules/language.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        did_you_mean = ['{} ({})'.format(i, phenny.ethno_data[i]) for i in iso if len(i) == 3]
+        response = "Try .iso639 for better results. Did you mean: " + ', '.join(did_you_mean) + "?"
+    else:
+        response = "That ISO code wasn't found. (Hint: use .iso639 for better results)"
+
+    phenny.say(response)
+
+lang.name = 'wikidata'
+lang.commands = ['wikidata']
+lang.example = '.wikidata khk'
```
Comment (jonorthwash) [inline modules/language.py]: Could this just be `.lg` ?  People don't want to know the backend they're using, they just want to see data.  Also, people are used to `.lg`.

Comment (scoopgracie) [reply modules/language.py]: See d5f55dc

**Comment by scoopgracie** [comment]
@jonorthwash how does d5f55dc look?

**Comment by scoopgracie** [comment]
@jonorthwash how is d5f55dc for the command? support lg but deprecate it in favor of .wikidata

**Comment by scoopgracie** [comment]
Also, we will need to deal with .iso639, which currently depends on ethnologue.

**Pull request merged**